changelog.txt

```diff
@@ -0,0 +1,2 @@
+fixed - fixes bug in Cloud Firestore emulator where rulesets with no-op writes would modify a document's updated_at timestamp
+changed - reduced lock contention in Cloud Firestore emulator during concurrent writes to a document
```

Comment:
```suggestion
fixed - Reduced lock contention in Cloud Firestore emulator during concurrent writes to a document.
```

"Changed" is for when the API surface changed, which I don't think is the case here.

Also, trying to use better sentences :)